terraform-old/core.tf

```diff
@@ -31,7 +31,7 @@ resource "packet_device" "core" {
   count            = 1 # nixops code doesn't support > 1 right now
   hostname         = "core-${count.index}.ewr1.nix.ci"
   plan             = "baremetal_0"
-  facility         = "ewr1"
+  facilities       = [ "ewr1" ]
```

Comment:
Should we just `rm -r terraform-old`? It's still present in git history if we ever need to refer to it again.

Reply:
:heavy_check_mark: